packages/cli/src/commands/init.ts

```diff
@@ -71,6 +81,41 @@ async function createApplicationHandler(config: ConfigService, answers: Answers)
   return createApplicationResponse.identifier;
 }
 
+async function raiseDemoDashboard() {
+  const notificationGroupResponse = await getNotificationGroup();
+
+  const template = buildTemplate(notificationGroupResponse.notificationGroupResponse[0]._id);
+
+  await createNotificationTemplates(template);
+}
+
+function buildTemplate(notificationGroupId: string): ICreateNotificationTemplateDto {
+  const redirectUrl = `${CLIENT_LOGIN_URL}?token={{token}}`;
+
+  const messages = [
+    {
+      type: ChannelTypeEnum.IN_APP,
+      content: 'Test content for <b>{{firstName}}</b>',
```

Comment:
```suggestion
      content: 'Welcome <b>{{firstName}}</b>! This your first notification, click on it to visit your live dashboard',
```

Reply:
Nice catch! I wanted to add 'click me' context as well so it will be more intuitive to the user!